Entries/Copy-X/Copy-X.md

```diff
@@ -0,0 +1,283 @@
+* [Reference Material](#reference-material) ([Alt.](#Reference%20Material))
+	* [Media](#media)
+	* [Additional Resources](#additional-resources) ([Alt.](#Additional%20Resources))
+	* [Footnotes](#footnotes)
+
+*** 
+
+## Categories: 
+
+* Optical Copy Protection
+
+***
+
+## Protection Features:
+
+* Ring protection
+* Dummy files
+* Disc check (Professional)
+* SoftICE Debugger Protection (Professional)
+
+***
+
+## DRML Entry Version: 
+
+***
+
+# DRM Information:
+
+## Names: 
+
+* copy-X[^CopyX_Original_Website]
+* copy-X - the ultimate Copy-Killer.[^CopyX_Original_Website]
+* copy-X audio[^CopyX_Original_Website]
+* copy-X light[^CopyX_Original_Website]
+* copy-X professional[^CopyX_Original_Website]
+* copy-X trial maker[^CopyX_Original_Website]
+
+[^CopyX_Original_Website]: [The original optimal media link for copy-X.](https://web.archive.org/web/20011016234742/http://www.optimal-online.de:80/product/copy_x.htm)
+
+***
+
+## Developers:
+
+* optimal media production (Until 2011)[^CopyX_Current_Website]
+* optimal media (Current)
+
+[^CopyX_Current_Website]: [The current optimal media website link.](https://www.optimal-media.com/)
+***
+
+## Overall Description:
+
+Ring protection copy protection scheme from german company optimal media[^CopyX_Original_Website]. As seen in Names, four different types seem to have been in use[^CopyX_Original_Website]. Note that it has not yet been confirmed whether light and professional correspond to the currently established distinct behaviors, but it seems to be the most likely conclusion.
+Had to be mastered at a specific optimal media production plant[^CopyX_2009_Document].
+
+[^CopyX_2009_Document]: [Optimal media document from 2009, seemingly for prospective customers.](http://press9.de/fileadmin/user_upload/PDF/copy_protection_july_09.pdf)
+***
+
+## Timeline: 
+
+#### 2000s
+
+* 2001?: Seems to come into existence about here.
+
+* 2009?: Some sort of overhaul occurs.
+
+***
+
+## Versions & Differences: 
+
+#### Tested:
+
+* copy-X light[^84759][^107929][^81628][^Kenny][^101786]
+* copy-X professional[^108150][^48393][^82475][^DE_EM2DX][^Petterson]
+
+[^84759]: [Black Cats and Pointed Hats (USA) (En,De)](http://redump.org/disc/84759/)
+[^107929]: [Robinson Crusoe (USA)](http://redump.org/disc/107929/)
+[^81628]: [Webmaster (USA))](http://redump.org/disc/81628/)
+[^108150]: [Case for TKKG, A - The Mystery of the Mayan Treasure (USA) (En,De)](http://redump.org/disc/108150/)
+[^48393]: [Emergency 4 - Global Fighters for Life (Germany) (Disc 1)](http://redump.org/disc/48393/)
+[^82475]: [Sluzhba 911 (Russia)][Remnants](http://redump.org/disc/82475/)
+[^DE_EM2DX]: [German Emergency 2 Deluxe](waiting on null to upload/)
+[^Petterson]: [Petterson und Findus - Hühnertanz und Katerzirkus](waiting on null to upload)
+[^Kenny]: [Kenny's Adventure](waiting on null to upload)
+[^101786]: [Max and the Magician (Europe) (En,Fr,De,Es)](http://redump.org/disc/101786/)
+
+#### copy-X light:
```

Comment:
Because the copy-X versions are part of the tested/untested categories, please make them a level 5 header instead of level 4, since that's the same as the tested/untested headers.

Reply:
Fixed.